### PR TITLE
Allow to point to a specific salt dir

### PR DESCRIPTION
**Pull request by ereslibre**
Improve development environment by creating a certificate authority,
salt-master with correct configuration for certificates to be remotely
signed and improve the start script to set up everything needed for the
development environment.

### CODE REVIEW

File: kubernetes/setup-environment [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    sleep 5
+done
+
+log "MariaDB container is ready"
+log "Creating database"
+docker exec -i $(docker ps | grep pharos-dashboard | awk '{print $1}') rake db:create &> /dev/null
+docker exec -i $(docker ps | grep pharos-dashboard | awk '{print $1}') bash -c "RAILS_ENV=test rake db:create" &> /dev/null
+log "Loading schema"
+docker exec -i $(docker ps | grep pharos-dashboard | awk '{print $1}') rake db:schema:load &> /dev/null
+docker exec -i $(docker ps | grep pharos-dashboard | awk '{print $1}') bash -c "RAILS_ENV=test rake db:schema:load" &> /dev/null
```
Comment (jimmykarily) [inline kubernetes/setup-environment]: Maybe we can remove the dummy Minions from seeds, add a default User there and run seeds. Right now we have to manually create a user in order to access the dashboard.

**Comment by jimmykarily** [comment]
Maybe add a something in the README to explain the SALT_DIR usage? Not any extensive documentation but we should leave such things completely undocumented.

File: kubernetes/start [hunk 2]
```diff
@@ -8,8 +8,26 @@ for template in $(ls manifest-templates)
...
+      sleep 5
+    else
+      salt_dir=$SALT_DIR
+    fi
+
+    sed -i -e "s#\${salt_dir}#$salt_dir#" manifests/$template
 done
 
-log "Launching kubelet"
+./cleanup
```
Comment (mssola) [inline kubernetes/start]: I'd prefer if you'd ask the user first before cleaning up. If the user doesn't want to do it, then we `exit 0`.

**Comment by ereslibre** [comment]
Updated with README instructions to use the `SALT_DIR`.

File: kubernetes/setup-environment (hunk 1 above)
Comment (ereslibre) [inline kubernetes/setup-environment]: Yes, I agree.

Comment (ereslibre) [inline kubernetes/setup-environment]: Fixed.

**Pull request merged**